standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java

```diff
@@ -30,4 +31,8 @@
    * Dumps platform state into logs
    */
   void dumpPlatformState();
+  /**
+   * Provides the raw config file from server startup
+   */
+  InputStream getPlatformConfiguration();
```

Comment:
Can you clarify?  Are you saying that we should mandate config file writers only use UTF-8?